flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,248 @@
+                          tvdb_id=parent_request.get('tvDbId'),
+                          imdb_id=parent_request.get('imdbId'),
+                          ombi_status=parent_request.get('status'),
+                          ombi_request_id=parent_request.get('id'))
+        elif config.get('type') == 'seasons':
+            log.debug('Season Number: %s', season.get('seasonNumber'))
+            tempSeries_id = 'S' + str(season.get('seasonNumber')).zfill(2)
+            entry = Entry(title=OmbiBase.generate_title(parent_request, config.get('strip_year')) + ' ' + tempSeries_id,
+                          url='http://www.imdb.com/title/' + parent_request.get('imdbId') +'/',
+                          series_name=OmbiBase.generate_title(parent_request, config.get('strip_year')),
+                          series_season=season.get('seasonNumber'),
+                          series_id=tempSeries_id,
+                          tvdb_id=parent_request.get('tvDbId'),
+                          imdb_id=parent_request.get('imdbId'),
+                          ombi_childrequest_id=child_request.get('id'),
+                          ombi_season_id=season.get('id'),
+                          ombi_status=parent_request.get('status'),
+                          ombi_request_id=parent_request.get('id'))
+        elif config.get('type') == 'episodes':
+            tempSeries_id = 'S' + str(season.get('seasonNumber')).zfill(2) + 'E' + str(episode.get('episodeNumber')).zfill(2)
+            entry = Entry(title=OmbiBase.generate_title(parent_request, config.get('strip_year')) + ' ' + tempSeries_id + ' ' + episode['title'],
+                          url=episode.get('url'),
+                          series_name=OmbiBase.generate_title(parent_request, config.get('strip_year')),
+                          series_season=season.get('seasonNumber'),
+                          series_episode=episode.get('episodeNumber'),
+                          series_id=tempSeries_id,
+                          tvdb_id=parent_request.get('tvDbId'),
+                          imdb_id=parent_request.get('imdbId'),
+                          ombi_request_id=parent_request.get('id'),
+                          ombi_childrequest_id=child_request.get('id'),
+                          ombi_season_id=season.get('id'),
+                          ombi_episode_id=episode.get('id'),
+                          ombi_approved=episode.get('approved'),
+                          ombi_available=episode.get('available'),
+                          ombi_requested=episode.get('requested'))
+        else:
+            raise plugin.PluginError('Error: Unknown list type %s.' % (config.get('type')))
+            
+        if config.get('type') in ['shows','seasons']:
+            # shows/seasons do not have approval or available status so return
+            return entry
+        elif config.get('only_approved') and not entry.get('ombi_approved'):
```

Comment:
These should be at the top. No need to create an entry that we won't use. The "fail fast" principle sort of applies here (it's not a failure though).